### PR TITLE
Improve search API Icinga alert documentation

### PR DESCRIPTION
**Pull request by ChrisBAshton**
See commits for details.

### CODE REVIEW

File: source/manual/alerts/runs-rake-task-search-api.html.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+owner_slack: "#govuk-2ndline"
+title: Runs a rake task on Search API that generates the sitemap files
```
Comment (bilbof) [inline source/manual/alerts/runs-rake-task-search-api.html.md]: Can I suggest "Generate sitemaps" as the title?

File: source/manual/alerts/runs-rake-task-search-api.html.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+parent: "/manual.html"
+layout: manual_layout
+section: Icinga alerts
+last_reviewed_on: 2020-01-14
+review_in: 6 months
+---
+
+At 2:30am every day, a [Jenkins job][jenkins] runs a rake task on search-api
```
Comment (bilbof) [inline source/manual/alerts/runs-rake-task-search-api.html.md]: The links to the job aren't working for me - looks like a markdown formatting issue.

Comment (sengi) [reply source/manual/alerts/runs-rake-task-search-api.html.md]: Ah yeah it's the parentheses around the URL in the footnote link.

File: source/manual/alerts/runs-rake-task-search-api.html.md (hunk 1 above)
Comment (ChrisBAshton) [reply source/manual/alerts/runs-rake-task-search-api.html.md]: This is an alert page documentation, so I've copied the alert as it appears in Icinga. I think this is the convention for all documentation under `manual/alerts`.

File: source/manual/alerts/runs-rake-task-search-api.html.md (hunk 2 above)
Comment (ChrisBAshton) [reply source/manual/alerts/runs-rake-task-search-api.html.md]: Whoops! Think it's the parentheses on line 25. Good spot!

File: source/manual/alerts/runs-rake-task-search-api.html.md (hunk 1 above)
Comment (bilbof) [reply source/manual/alerts/runs-rake-task-search-api.html.md]: OK, thank you for explaining!

**Pull request merged**